server/backend/influxdb.go

```diff
@@ -0,0 +1,39 @@
+package backend
+
+import (
+	"context"
+	"errors"
+	"github.com/TUM-Dev/Campus-Backend/server/backend/influx"
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
+	"os"
+)
+
+const (
+	// InfluxBatchSize is the number of points that are sent to influxdb at once
+	// This is to reduce the number of requests to the database
+	InfluxBatchSize = 1_000
```

Comment:
1000 seems like a lot, how many datapoints do you expect in production (e.g. per minute)?